modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryMetadata.java

```diff
@@ -387,4 +402,11 @@ public Map<String, Integer> enumMap() {
     @Override public String toString() {
         return S.toString(BinaryMetadata.class, this);
     }
+
+    /**
+     * @return Schema IDs registered for this type.
+     */
+    public Set<Integer> schemaIds() {
+        return Collections.unmodifiableSet(schemaIds);
```

Comment:
just schemaIds. Let's avoid creating useless wrappers. This is internal code, users can't change the collection and Ignite developers will not do it (we trust committers)